modules/flowable-cxf/src/main/java/org/flowable/engine/impl/webservice/WSDLImporter.java

```diff
@@ -211,14 +211,8 @@ protected void importStructure(Mapping mapping) {
                             isFieldParameterTypeNeestedClass = true;
                         }
                     } while (!isFieldParameterTypeNeestedClass);
-                    if (isFieldParameterTypeNeestedClass) {
-                        // The parameter type is a nested class
-                        fieldParameterClass = ReflectUtil
-                                .loadClass(theClass.erasure().fullName() + "$" + fieldParameterType.name());
-                    } else {
-                        // The parameter type is not a nested class
-                        fieldParameterClass = ReflectUtil.loadClass(fieldParameterType.erasure().fullName());
-                    }
+                    // The parameter type is a nested class
+                    fieldParameterClass = ReflectUtil.loadClass(theClass.erasure().fullName() + "$" + fieldParameterType.name());
```

Comment:
You copied the wrong if branch, that's why it was failing. It should be the one with the "The parameter type is not a nested class".